Dockerfile.autotune

```diff
@@ -16,15 +16,21 @@
 ##########################################################
 #            Build Docker Image
 ##########################################################
-FROM maven:3-eclipse-temurin-17 as mvnbuild-jdk17
-
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7 as mvnbuild-jdk17
 ARG USER=autotune
 ARG AUTOTUNE_HOME=/home/$USER
 
-# Install any additional packages that a typical developer in the team needs.
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends git vim \
-    && rm -rf /var/lib/apt/lists/*
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-17-openjdk-devel \
+     && microdnf install tar gzip java-17-openjdk-jmods binutils git vim \
+     && microdnf clean all 
+
+RUN microdnf install tar gzip  java-17-openjdk-jmods binutils
```

Comment:
Thanks for the update @xrajesh, Looks like this line can now be removed?

Reply:
Thank you @dinogun , I also updated the Maven version.

Reply:
@dinogun Updated maven to 3.9.0 .. Please check.